modin/engines/dask/pandas_on_dask/frame/partition.py

```diff
@@ -178,7 +177,7 @@ def mask(self, row_indices, col_indices):
             A new ``PandasOnDaskFramePartition`` object.
         """
         new_obj = super().mask(row_indices, col_indices)
-        client = _get_global_client()
+        client = default_client()
```

Comment:
I wonder why it was different from other functions before - why this one used `_get_global_client` while others used `get_client`...

Reply:
I guess it was done by different people that had different knowledge on API in Dask. We could ask it from Devin.
cc @devin-petersohn

Reply:
At one point everything was `_get_global_client` because it didn't throw an exception if there wasn't an existing client. This change is good.